### PR TITLE
PDS API Postman collection

### PR DESCRIPTION
**Pull request by gjugpal**
## Summary
* Routine Change
Uploaded the PDS API Postman collection, which is the Regression pack of Automated tests

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: tests/e2e/smoke.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+						"key": "NHSD-Identity-JWT",
+						"value": "{{jwt}}"
+					},
+					{
+						"key": "NHSD-Session-URID",
+						"value": "1234567890"
+					}
+				],
+				"url": {
+					"raw": "{{environment}}/Patient/5900018512",
```
Comment (alexhawdon) [inline tests/e2e/smoke.json:51]: Why doesn't this patient start with a 9?

Comment (Sourceless) [reply tests/e2e/smoke.json:51]: @gjugpal

Comment (gjugpal) [reply tests/e2e/smoke.json:51]: This is just a random Patient that I created on VEIT07. I'll be adding more tests to this collection in due course.

Comment (alexhawdon) [reply tests/e2e/smoke.json:51]: @gjugpal  the convention is that test NHS IDs all start with a 9. One that doesn't makes us nervous!

Comment (gjugpal) [reply tests/e2e/smoke.json:51]: @alexhawdon I wasn't aware of that, the DSA tool which we use to create/amend patients, auto generates NHS numbers on create. To confirm this is a Patients NHS number not a clinicians NHS Id used to login to the client.

Comment (Sourceless) [reply tests/e2e/smoke.json:51]: The only spine env this regression pack will run against is VEIT07 so we should be safe

Comment (Sourceless) [reply tests/e2e/smoke.json:51]: Ideally if we could hit a healthcheck endpoint that would be better

Comment (Sourceless) [reply tests/e2e/smoke.json]: I've repointed it to `/healthcheck`

**Pull request merged**